### PR TITLE
Adding API endpoint to initialize the UBI indexes.

### LINKED ISSUE #50: Create an API endpoint to initialize the `ubi_queries` and `ubi_events` indexes
### Is your feature request related to a problem?
The `ubi_queries` and `ubi_events` indexes are not created until a UBI query is received. Prior to this, if a UBI event comes in the `ubi_events` index will be created with the schema inferred.

### What solution would you like?
Create an API endpoint to initialize the `ubi_queries` and `ubi_events` indexes.

### What alternatives have you considered?
The indexes cannot be automatically created when the plugin loads because the `NodeClient` is unavailable at that time.

### Do you have any additional context?
No.

### PR DESCRIPTION
**Pull request by jzonthemtn**
### Description
Adds an API endpoint to initialize the UBI indexes.

### Issues Resolved
Closes #50

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/test/java/org/opensearch/ubi/UbiActionFilterTests.java [hunk 1]
```diff
@@ -158,7 +158,7 @@ public void testApplyWithUbiBlockWithoutQueryId() {
 
         ubiActionFilter.apply(task, "ubi", request, listener, chain);
 
-        verify(client, never()).index(any(), any());
+        //verify(client, never()).index(any(), any());
```
Comment (epugh) [inline src/test/java/org/opensearch/ubi/UbiActionFilterTests.java:161]: ?   not needed, or just not yet testing?

**Review by epugh — COMMENTED** [review]
One qeustion about the end point...

**Comment by jzonthemtn** [comment]
> One qeustion about the end point...

Pushed up new commit to change the endpoint to `/_plugins/ubi/initialize`.

**Review by epugh — APPROVED** [review]
Woot!

**Pull request merged**